bin/i18n/sync-in.rb

```diff
@@ -647,12 +647,14 @@ def localize_markdown_content
     hourofcode/unplugged-conditionals-with-cards.md.partial
     international/about.md.partial
     poetry.md.partial
+    ../views/hoc2022_create_activities.md.partial
```

Comment:
Add the other two files:
../views/hoc2022_play_activities.md.partial
../views/hoc2022_explore_activities.md.partial

Reply:
shoot yes. thank you!

Reply:
Done!